File: matiasmasso/Mat.NET/DTO/DTOQtyDto.cs
namespace DTO
{
    public class DTOQtyDto
    {
        public int Qty { get; set; }
        public decimal Dto { get; set; }
        public int FreeUnits { get; set; }

        public DTOQtyDto(int iQty = 0, decimal DcDto = 0, int iFreeUnits = 0) : base()
        {
            Qty = iQty;
            Dto = DcDto;
            FreeUnits = iFreeUnits;
        }
    }
}

File: matiasmasso/Mat.NET/DT2/DTOQtyDto.vb
Public Class DTOQtyDto
    Property Qty As Integer
    Property Dto As Decimal
    Property FreeUnits As Integer

    Public Sub New(Optional ByVal iQty As Integer = 0, Optional ByVal DcDto As Decimal = 0, Optional ByVal iFreeUnits As Integer = 0)
        MyBase.New()
        _Qty = iQty
        _Dto = DcDto
        _FreeUnits = iFreeUnits
    End Sub
End Class
